extensions/amp-analytics/0.1/requests.js

```diff
@@ -85,9 +87,6 @@ export class RequestHandler {
     /** @private {?Promise<string>} */
     this.baseUrlTemplatePromise_ = null;
 
-    /** @private {!Array<!Promise<string>>}*/
-    this.extraUrlParamsPromise_ = [];
```

Comment:
this is removed. all data are always stored in `this.batchSegmentPromises_`. and we provide a `defaultBatchPlugin` function if none is specified